src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py

```diff
@@ -386,7 +386,6 @@ def _evaluate(
         evaluator_config = {}
     evaluator_config = _process_evaluator_config(evaluator_config)
     _validate_columns(input_data_df, evaluators, target, evaluator_config)
-
     # Target Run

```

Comment:
I believe PFClient initialzation will throw MissingAzurePackage if pf-azure is not installed, we should catch the exception here and print the warning message with the instruction to install the package.

Also, please restore the blank line that was removed.

Reply:
Added the try-catch and corresponding test.